src/connection.ts

```diff
@@ -131,6 +135,7 @@ export class Connection extends EventEmitter {
     this.connectionTag = opts.connectionTag
     this.maxStreamId = 2 * (opts.maxRemoteStreams || DEFAULT_MAX_REMOTE_STREAMS)
     this.maxBufferedData = opts.connectionBufferSize || MAX_DATA_SIZE * 2
+    this.minExchangeRatePrecision = opts.minExchangeRatePrecision || DEFAULT_MINIMUM_EXCHANGE_RATE_PRECISION
```

Comment:
You should be able to set it to 0

Reply:
I don't think the precision can be 0, even if you put in 0 for a number the precision will come out as 1 so they can just set it to 1.

Reply:
Theoretically you could be okay with sending money to deliver 0-amount packets to the destination. It's a nitpick but in this case I think it's better to do `typeof opts.minExchangeRatePrecision === 'number' ? opts.minExchangeRatePrecision : DEFAULT...`

Reply:
The precision function can't return zero, though; 0 has a precision of 1

Reply:
:+1: true

Reply:
Left this as is since as Ben mentioned the precision will always be at least 1.